spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -135,6 +135,15 @@ def get_parser():
         action="help",
         help="Show this help message and exit."
     )
+    optional.add_argument(
+        '-s-template-id',
+        metavar=Metavar.int,
+        type=int,
+        help="Segmentation from the template to associate with the flag '-s'. Typically, the spinal cord segmentation "
+             "is used, but if it is available, the white matter mask would produced better registration results. The ID"
+             "is indicated in the file 'template/info_label.txt' of the template indicated by the flag '-t'.",
```

Comment:
```suggestion
        help="Segmentation from the template to be used by flag '-s'. By default, the spinal cord segmentation "
             "is used, but if available, a different segmentation could produced better registration results. The ID"
             "is an integer indicated in the file 'template/info_label.txt' of the template indicated by the flag '-t'.",
```

Reply:
no that's not it. "-s" is the seg from the subject, while this new flag "-s-template-id" is the seg from the template. that's why i used the term "associate with"-- could you suggest something clearer maybe? if not we stick with associate